Week2/homework/map-filter.js

```diff
@@ -12,4 +12,29 @@ console.log(doubleOddNumbers(myNumbers));
 module.exports = {
   myNumbers,
   doubleOddNumbers,
+};  */
+
+'use strict';
+
+function doubleOddNumbers(numbers) {
+
+  numbers = numbers.filter(num => num % 2 === 1).map(num => num * 2);
```

Comment:
Nice job! But there seems to be many blank lines above and below. Can you delete?